mail_prevent_bounce_loop/README.rst

```diff
@@ -0,0 +1,110 @@
+========================
+Mail Prevent Bounce Loop
+========================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:a2c5d232ac1c492e303d58e122afb9f061be446596049b2431a4105fd481f04e
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsocial-lightgray.png?logo=github
+    :target: https://github.com/OCA/social/tree/17.0/mail_prevent_bounce_loop
+    :alt: OCA/social
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/social-17-0/social-17-0-mail_prevent_bounce_loop
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/social&target_branch=17.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module helps to prevent infinite loop between odoo and
+autoresponder, which is currently raised in
+`here <https://github.com/odoo/odoo/issues/132666>`__
+
+-  Case 1: odoo sends an invoice email to customer mail address,
+   autoresponder sends auto-reply mail (without enough information for
+   odoo to detect it as an auto-reply: headers
+   `-loop-detection-bounce-email@/-loop-detection-bounce-email <https://github.com/odoo/odoo/blob/9be0c5348bfeb338bcba95b2a9c01e0d7dd14306/addons/mail/models/mail_thread.py#L1373>`__).
+   Odoo receives and send another one, resulting ping-pong situation.
+-  Case 2: a cron, Notification: Send scheduled message notifications.
+   When odoo receives automatic reply, the cron (running by default
+   every hour) sends notifications to all involved followers of the
+   mail, which then leads to the above issue
+
+Users can avoid the issue by disabling bounce mail at partner level.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To never send a bounce email to a specific email address, you can tick
+no_bounce_email as below:
+
+|no_bounce_email|
+
+To never send email notification when receiving auto-reply email from a
+specific email address identified as an auto-reply address, you can tick
+is_automatic_reply_address as below:
+
+|is_automatic_reply|
+
```

Comment:
in odoo this seems to be not well displayed:

![image](https://github.com/user-attachments/assets/fff47936-475e-4623-b8d7-982834c5c822)